less/home-page.less

```diff
@@ -45,10 +53,18 @@
     padding: 50px 60px;
     display: flex;
     justify-content: center;
-
+    @media @phone {
+      flex-direction: column-reverse;
+      margin-top: 10px;
+      padding: 10px 5%;
+    }
     .walk-text {
-      width: 50%;
-
+      width: 100%;
+      
+      @media @phone {
+        width: 100%;
```

Comment:
same thing here.  no need for width in the media query after setting the same thing on the element